incubator/busbar/templates/busbar-web-deployment.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "busbar.fullname" . }}-web
+  labels:
+    app: {{ template "busbar.name" . }}
+    component: {{ template "busbar.name" . }}-web
```

Comment:
Components should not use the name template. that's already in the `app` label.

`component: web`